File: client/src/app/core/data/activity-list.ts
export const ActivityList: ActivityBase[] = [
    {
        id: 0,
        description: 'Weightlifting',
        value: 2
    },
    {
        id: 1,
        description: 'Yoga/Stretching',
        value: 2
    },
    {
        id: 2,
        description: 'Running/Cardio',
        value: 2
    },
    {
        id: 3,
        description: 'Other Conditioning',
        value: 2
    },
    {
        id: 4,
        description: 'Practice',
        value: 1
    },
    {
        id: 5,
        description: 'Recreational Activities',
        value: 0.5
    },
];

export interface ActivityBase {
    id: number;
    description: string;
    value: number;
}

File: api/api/Models/ActivityList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace api.Models
{
    public class Activities
    {
        public List<ActivityItem> ActivityList {get;set;}

        public Activities()
        {
            this.ActivityList = new List<ActivityItem>() {
                { new ActivityItem() {Id = 0, Description="Weightlifting", Value=4}},
                { new ActivityItem() {Id = 1, Description="Yoga/Stretching", Value=4}},
                { new ActivityItem() {Id = 2, Description="Running/Cardio", Value=4}},
                { new ActivityItem() {Id = 3, Description="Other Conditioning", Value=4}},
                { new ActivityItem() {Id = 4, Description="Practice", Value=2}},
                { new ActivityItem() {Id = 5, Description="Recreational Activities", Value=1}}
            };
        }
    }

    public class ActivityItem
    {
        public int Id {get;set;}
        public string Description {get;set;}
        public decimal Value {get;set;}
    }
}
